api/UIEvent.json

```diff
@@ -462,10 +462,10 @@
               }
             ],
             "safari": {
-              "version_added": "≤4"
+              "version_added": "3"
```

Comment:
This is `which` and wasn't in https://github.com/WebKit/WebKit/blob/7659cc58c0a8c7c04bc854486cb8355a13c008dd/WebCore/khtml/ecma/kjs_events.cpp, can you research when it was added?

Reply:
Looks like this was added in https://github.com/WebKit/WebKit/commit/efe8bf5d4aa0c3ccc29f8f459d7b584812e5f223 which predates Safari 1!